test/letters/components/DownloadLetterLink.unit.spec.js

```diff
@@ -3,43 +3,33 @@ import SkinDeep from 'skin-deep';
 import { expect } from 'chai';
 import sinon from 'sinon';
 
-import DownloadLetterLink from '../../../src/js/letters/components/DownloadLetterLink.jsx';
+import { DownloadLetterLink } from '../../../src/js/letters/components/DownloadLetterLink.jsx';
 
 const defaultProps = {
   letterName: 'Commissary Letter',
   letterType: 'commissary'
 };
 
-
 let oldWindow;
 let oldFetch;
 
+// TODO: fix this warning and improve test coverage for various scenarios:
```

Comment:
This still needs a ton of work but just trying to get something functional for usability tests tomorrow.